js/api-wrapper.js

```diff
@@ -79,22 +78,22 @@ function closeTask (taskId) {
   if (taskId === currentTask.id) {
     // the current task was destroyed, find another task to switch to
 
-    if (tasks.get().length === 0) {
+    if (tasks.getLength() === 0) {
```

Comment:
The equivalement method for tabs is `tabs.count()`, we probably want to use the same name for both (whichever one you want to use is fine with me).

Reply:
Of course! Since `getLength()` does in fact `this.tasks.length`, I think its name is quite appropriate. 
So I'd suggest that the tabs' equivalent is changed to this one - and that would probably happen when the tabs prototype goes through the same type of change as this PR does to the task list.